schema/deploy/shipping/views.sql

```diff
@@ -669,4 +669,39 @@ comment on view shipping.sample_with_best_available_encounter_data_v1 is
     'Version 1 of view of warehoused samples and their best available encounter date and site details important for metrics calculations';
 
 
+create or replace view shipping.return_results_v2 as
+
+    select barcode,
+           case
+             when sample_id is null then 'notReceived'
+             when sample_id is not null and count(present) = 0 then 'processing'
+             when count(present) > 0 then 'complete'
+           end as status,
+           array_agg(distinct organism::text)
+            filter (where present and organism is not null)  as organisms_present,
+           array_agg(distinct organism::text)
+            filter (where not present and organism is not null) as organisms_absent
+
+      from warehouse.identifier
+      join warehouse.identifier_set using (identifier_set_id)
+      left join warehouse.sample on uuid::text = sample.collection_identifier
+      left join warehouse.encounter using (encounter_id)
+      left join shipping.presence_absence_result_v2 on sample.identifier = presence_absence_result_v2.sample
+
+    --These are all past and current collection identifier sets not including self-test
+    where identifier_set.name in ('collections-kiosks',
+                                  'collections-swab&send')
```

Comment:
I think this should exclude kiosks (which for sure aren't running and it also seems don't have updated consent) and include `collections-self-test`, which shares the same REDCap project as swab and send now.

Reply:
Perhaps swab & send asymptomatic should also be added, too?

Reply:
If the consent was updated there at the same time, then yes.

Reply:
It doesn't appear to be updated, yet. So I will leave it off for now.